### PR TITLE
Add Bitforex

### PR DESCRIPTION
**Pull request by 40504**
Bitforex

### CODE REVIEW

File: _data/cryptocurrencies.yml [hunk 1]
```diff
@@ -69,6 +69,12 @@ websites:
     regions:
       - jp
 
+  - name: Bitforex
+    url: https://bitforex.com
+    img: bitforex.svg
+    tfa:
+      - totp
```
Comment (phallobst) [inline _data/cryptocurrencies.yml:76]:
```suggestion
      - totp
      - sms
    doc: support.bitforex.com/hc/en-us/articles/360006825152
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by phallobst — APPROVED** [review]

File: _data/cryptocurrencies.yml [hunk 2]
```diff
@@ -69,6 +69,14 @@ websites:
...
+      - sms
+    doc: support.bitforex.com/hc/en-us/articles/360006825152
```
Comment (ApCoder123) [inline _data/cryptocurrencies.yml]:
```suggestion
    doc: https://support.bitforex.com/hc/en-us/articles/360006825152
```

File: img/cryptocurrencies/bitforex.svg [hunk 3]
```diff
@@ -0,0 +1 @@
..." y2="68.988%"><stop stop-color="#00ffc8" offset="0"/><stop stop-color="#00758c" offset="1"/></linearGradient><linearGradient id="B"><stop stop-color="#00ffc8" offset="0"/><stop stop-color="#00959a" offset="1"/></linearGradient><linearGradient id="C" y1="30.719%" y2="30.719%"><stop stop-color="#00ffc8" offset="0"/><stop stop-color="#00bbab" offset="1"/></linearGradient></defs><g fill="none" fill-rule="evenodd"><path fill="#fff" opacity="0" d="M0 0h90v90H0z"/><g transform="translate(0 18)" fill-rule="nonzero"><path d="M56.355 32.2c0 6.405-5.22 11.616-11.635 11.616S33.084 38.605 33.084 32.2V20.043h23.271V32.2zM67.106 9.493V-.001H56.355v20.043H33.084V-.001H22.333V32.2c0 12.323 10.043 22.348 22.388 22.348S67.107 44.522 67.107 32.2V9.493z" fill="url(#A)"/><path d="M78.327 0c0 5.273-4.266 9.494-9.504 9.494H20.294c-5.244 0-9.51-4.259-9.51-9.494H0c0 11.517 8.898 20.043 20.428 20.043h48.531c11.536 0 20.211-8.49 20.211-20.043H78.328z" fill="url(#B)"/><path d="M56.355 32.2c0 6.405-5.22 11.616-11.635 11.616S33.084 38.605 33.084 32.2V20.043h23.271V32.2zm0-12.157H33.084V0H22.333v32.201c0 12.323 10.043 22.348 22.388 22.348s22.386-10.026 22.386-22.348V20.044H56.356z" fill="url(#C)"/></g></g></svg>
```
Comment (ApCoder123) [inline img/cryptocurrencies/bitforex.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 90 90"><defs><linearGradient id="A" x1="19.341%" x2="153.77%" y1="30.719%" y2="68.988%"><stop stop-color="#00ffc8" offset="0"/><stop stop-color="#00758c" offset="1"/></linearGradient><linearGradient id="B"><stop stop-color="#00ffc8" offset="0"/><stop stop-color="#00959a" offset="1"/></linearGradient><linearGradient id="C" y1="30.719%" y2="30.719%"><stop stop-color="#00ffc8" offset="0"/><stop stop-color="#00bbab" offset="1"/></linearGradient></defs><g fill="none" fill-rule="evenodd"><path fill="#fff" opacity="0" d="M0 0h90v90H0z"/><g transform="translate(0 18)" fill-rule="nonzero"><path d="M56.355 32.2c0 6.405-5.22 11.616-11.635 11.616S33.084 38.605 33.084 32.2V20.043h23.271V32.2zM67.106 9.493V-.001H56.355v20.043H33.084V-.001H22.333V32.2c0 12.323 10.043 22.348 22.388 22.348S67.107 44.522 67.107 32.2V9.493z" fill="url(#A)"/><path d="M78.327 0c0 5.273-4.266 9.494-9.504 9.494H20.294c-5.244 0-9.51-4.259-9.51-9.494H0c0 11.517 8.898 20.043 20.428 20.043h48.531c11.536 0 20.211-8.49 20.211-20.043H78.328z" fill="url(#B)"/><path d="M56.355 32.2c0 6.405-5.22 11.616-11.635 11.616S33.084 38.605 33.084 32.2V20.043h23.271V32.2zm0-12.157H33.084V0H22.333v32.201c0 12.323 10.043 22.348 22.388 22.348s22.386-10.026 22.386-22.348V20.044H56.356z" fill="url(#C)"/></g></g></svg>
```

The `xmlns:v` attribute is unnecessary

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

File: _data/cryptocurrencies.yml [hunk 4]
```diff
@@ -69,6 +69,14 @@ websites:
     regions:
       - jp
 
+  - name: Bitforex
```
Comment (ApCoder123) [inline _data/cryptocurrencies.yml]:
```suggestion
  - name: BitForex
```
It is capitalised like this on their site:

> BitForex | Safest Bitcoin Trading Platform

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

File: _data/cryptocurrencies.yml (hunk 4 above)
Comment (40504) [reply _data/cryptocurrencies.yml]: Done ✅

**Review by ApCoder123 — APPROVED** [review]

**Pull request merged**